# Task
Perform a hypothesis test to determine if the weekly operating costs for a restaurant are significantly higher than the theoretical mean, calculated using the formula W = $1,000 + $5X (where X = 600 units). Use a significance level of 5% for this test. The process should include calculating the theoretical mean, stating the null and alternative hypotheses, calculating the test statistic, determining the critical value, making a decision, and drawing a conclusion about the restaurant owners' claim.

## Calculate Theoretical Mean Weekly Cost

### Subtask:
Calculate the theoretical mean weekly operating cost (μ) using the given formula W = $1,000 + $5X, where X is 600 units.


In [5]:
X = 600
W = 1000 + 5 * X
print(f"The theoretical mean weekly operating cost (μ) is: ${W}")

The theoretical mean weekly operating cost (μ) is: $4000


## State Hypotheses


*   **Null Hypothesis (H0):** The true mean weekly operating cost (μ) is less than or equal to the theoretical mean.
    H0: μ ≤ $4000

*   **Alternative Hypothesis (H1):** The true mean weekly operating cost (μ) is significantly greater than the theoretical mean.
    H1: μ > $4000

## Calculate Test Statistic

### Subtask:
Calculate the test statistic (t) using the provided formula and given values for sample mean, theoretical mean, standard deviation (sigma = 5*25), and sample size.


In [6]:
import math

sample_mean = 4100
sample_size = 30
population_mean = W # W was calculated in the previous step and is 4000
standard_deviation = 5 * 25 # sigma = 5 * 25

standard_error = standard_deviation / math.sqrt(sample_size)
test_statistic = (sample_mean - population_mean) / standard_error

print(f"The sample mean is: ${sample_mean}")
print(f"The population mean (theoretical mean W) is: ${population_mean}")
print(f"The standard deviation (sigma) is: ${standard_deviation}")
print(f"The sample size is: {sample_size}")
print(f"The calculated Standard Error of the Mean is: {standard_error:.2f}")
print(f"The calculated Test Statistic (t) is: {test_statistic:.2f}")

The sample mean is: $4100
The population mean (theoretical mean W) is: $4000
The standard deviation (sigma) is: $125
The sample size is: 30
The calculated Standard Error of the Mean is: 22.82
The calculated Test Statistic (t) is: 4.38


## Determine Critical Value

### Subtask:
Determine the critical value from the standard normal (Z) distribution table for an alpha level of 5% (α = 0.05) for a one-tailed test (since the claim is 'higher').


In [7]:
from scipy.stats import norm

alpha = 0.05
critical_z_value = norm.ppf(1 - alpha)

print(f"The significance level (alpha) is: {alpha}")
print(f"The critical Z-value for a one-tailed test at alpha = {alpha} is: {critical_z_value:.2f}")

The significance level (alpha) is: 0.05
The critical Z-value for a one-tailed test at alpha = 0.05 is: 1.64


## Make Decision and Conclude

### Subtask:
Compare the calculated test statistic with the critical value to make a decision regarding the null hypothesis and draw a conclusion about the restaurant owners' claim.


In [8]:
test_statistic = 4.381780460041329
critical_z_value = 1.6448536269514722

print(f"Test Statistic: {test_statistic:.2f}")
print(f"Critical Z-value: {critical_z_value:.2f}")

if test_statistic > critical_z_value:
    decision = "Reject the null hypothesis (H0)."
    conclusion = "There is statistically significant evidence to support the claim that the weekly operating costs for the restaurant are significantly higher than the theoretical mean."
else:
    decision = "Fail to reject the null hypothesis (H0)."
    conclusion = "There is not enough evidence to support the claim that the weekly operating costs for the restaurant are significantly higher than the theoretical mean."

print(f"\nDecision: {decision}")
print(f"Conclusion: {conclusion}")

Test Statistic: 4.38
Critical Z-value: 1.64

Decision: Reject the null hypothesis (H0).
Conclusion: There is statistically significant evidence to support the claim that the weekly operating costs for the restaurant are significantly higher than the theoretical mean.


## Final Task

### Subtask:
Summarize the findings from the hypothesis test, detailing whether there is strong evidence to support the claim that weekly operating costs are higher.


## Summary:
Yes, there is statistically significant evidence to support the claim that the weekly operating costs for the restaurant are significantly higher than the theoretical mean.

### Data Analysis Key Findings
*   The theoretical mean weekly operating cost ($\mu$) was calculated to be \$4000, based on the formula W = \$1,000 + \$5X with X = 600 units.
*   The null hypothesis (H0) was set as $\mu \le \$4000$, and the alternative hypothesis (H1) was $\mu > \$4000$, indicating a one-tailed test.
*   Using a sample mean of \$4100, a sample size of 30, and a standard deviation of \$125, the calculated test statistic was 4.38.
*   For a one-tailed test with a significance level of 5% ($\alpha = 0.05$), the critical Z-value was determined to be 1.64.
*   Since the calculated test statistic (4.38) is greater than the critical Z-value (1.64), the null hypothesis was rejected. This indicates strong statistical evidence.

### Insights or Next Steps
*   The restaurant's actual weekly operating costs appear to be genuinely higher than the initial theoretical model predicted, suggesting a need to investigate the factors contributing to this increase.
*   Further analysis could involve identifying specific cost drivers that have deviated from the theoretical model and assessing their impact on profitability.
